RxJavaTemplate/app/src/main/java/co/nimblehq/rxjava/ui/screens/home/HomeFragment.kt

```diff
@@ -42,13 +46,13 @@ class HomeFragment : BaseFragment() {
     }
 
     private fun bindData(data: Data) {
-        tvContent.text = data.content
-        ivPreview.loadImage(data.imageUrl)
+        binding.tvContent.text = data.content
+        binding.ivPreview.loadImage(data.imageUrl)
```

Comment:
can refactor a bit by using `binding` with `run` 🙏
```suggestion
        binding.run {
            tvContent.text = data.content
            ivPreview.loadImage(data.imageUrl)
        }
```